### PR TITLE
[Ch  #166240813] deploy base application to heroku

### PR DESCRIPTION
**Pull request by alainburindi**
#### What does this PR do?
Deploy the base appplication to heroku
#### How should this be manually tested?
for development process access the endpoints using this link https://savage-rangers-ah-staging.herokuapp.com/
for deployment process use this link https://savage-rangers-ah.herokuapp.com/
#### What are the relevant pivotal tracker stories?
[#166240813](https://www.pivotaltracker.com/story/show/166240813)

### CODE REVIEW

**Comment by bdushimi** [comment]
Please follow the PR naming convention:
#[STORY_ID] [Story description]

**Review by bdushimi — CHANGES_REQUESTED** [review]
Please follow the PR naming convention:
#[STORY_ID] [Story description]

**Comment by bdushimi** [comment]
This link https://savage-rangers-ah.herokuapp.com/ is not up!

**Comment by alainburindi** [comment]
> Please follow the PR naming convention:
> #[STORY_ID] [Story description]

done

**Comment by alainburindi** [comment]
> This link https://savage-rangers-ah.herokuapp.com/ is not up!

It is waiting for the deployment process from Tarvis-ci

**Review by truestbyheart — COMMENTED** [review]
Can you remove the callback function on ```app.listen```

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+// routes (to be replaced)
+app.use('*', ( req, res, next ) => {
+  return res.status(200).json({
+    message : 'Welcome',
+    purpose : 'heroku test',
+    wish : 'See you soon',
+  });
+});
+
+app.listen( port, () =>{});
```
Comment (bdushimi) [inline src/index.js]: Usually, you want to console/print a meaningful message in this callback function... something like " app listening on port ${port}!` "

**Review by bdushimi — CHANGES_REQUESTED** [review]

**Comment by bdushimi** [comment]
I do not see any configurations that have something to do with deploying the app on Heroku in this PR.

**Comment by bdushimi** [comment]
Testing re-opening this PR

**Pull request merged**